extensions/linz/README.md

```diff
@@ -61,18 +62,19 @@ See [ISO/IEC 13249-3:2016(en)](https://www.iso.org/obp/ui/#!iso:std:60343:en) fo
+| summaries/updated/maximum                                                                                                       | date-time                                     | **REQUIRED**. Latest [asset updated value](#asset-fields), in UTC.                                                                                                                                                                                                                                                                               |
+| summaries/linz:geospatial_type                                                                                                  | \[string]                                     | **REQUIRED**. A general description of the type of content that can be found in the dataset. See the [list of accepted geospatial types](#geospatial-types).                                                                                                                                                                                     |
+| linz:lifecycle                                                                                                                  | string                                        | **REQUIRED**. Lifecycle Status of Collection. Must be one of `under development`, `preview`, `ongoing`, `completed`, `deprecated`.                                                                                                                                                                                                               |
+| linz:providers                                                                                                                  | [LINZ Provider Object](#linz-provider-object) | **REQUIRED**.                                                                                                                                                                                                                                                                                                                                    |
+| providers                                                                                                                       | [Provider Object](#provider-object)           | **REQUIRED**.                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
And here too?

`See [Providers Object](add link).`

Reply:
There is already a link though.

Reply:
See https://github.com/linz/stac/pull/138